ci/builders/mac_ios_engine.json

```diff
@@ -378,24 +331,9 @@
             "destination": "ios-release/Flutter.dSYM.zip",
             "realm": "production"
         },
-        {
```

Comment:
No needed anymore?

Reply:
Yeah, so i moved the Flutter.framework for extension_safe into the responding existing zips. In below python change
For example
debug extension_safe Flutter.framework is now moved under ios-debug/extension_safe